test/js/node/fs/abort-signal-leak-read-write-file-fixture.ts

```diff
@@ -0,0 +1,32 @@
+import fs from "fs";
+import { join } from "path";
+import { tmpdirSync } from "harness";
+
+const tmpdir = tmpdirSync();
+
+for (let i = 0; i < 100_000; i++) {
```

Comment:
This test should check heapStats to verify that the number of `AbortSignal` in objectTypeCounts is < 10.

Reply:
updated